boards/arm/v2m_musca/v2m_musca_nonsecure_defconfig

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2018 Linaro Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_ARM=y
+CONFIG_SOC_FAMILY_ARM=y
+CONFIG_SOC_SERIES_MUSCA=y
+CONFIG_SOC_V2M_MUSCA_A=y
+CONFIG_BOARD_MUSCA_A_NONSECURE=y
```

Comment:
See comment, above, in the definition of this option.

Reply:
going to remove this file, per our discussion

Reply:
I've found that I cannot remove this as the build system cannot adequately determine the DTS file.  We either have to leave the -DBOARD=v2m_musca_nonsecure as a build option and the only way to get a nonsecure image, or we come up with another way of determining the appropriate base DTS file to compile in the cmake scripts based off of some selection process.